Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -191,6 +197,14 @@ def check_pull_request_args(cls, repository, args):
 
     @classmethod
     def pull_request_branch_point(cls, repository, args, **kwargs):
+        if args.redact and len(repository.source_remotes()) <= 1:
```

Comment:
If repository.source_remotes() is 1, it can't be security one?

Reply:
Not if your checkout is set up correctly. `source_remotes` are defined by the project, not the user. And the concept of "redacting" a PR doesn't make any sense if the primary repository is an acceptable place for such PRs. In that case, the project simply would not define rules for redacting commits and all commits would be made against `origin`.